lib/player.rb

```diff
@@ -1,7 +1,4 @@
 module TicTacToe
-  
-  MARKERS = ["X", "O"]
-
   class Player
 
```

Comment:
I think breaking them out is a good idea and is more explicit. The main reason behind placing it in the `Constants` file and not in `Player` class is because `UserInterface` also needs access to the `HumanPlayer` marker.